custom-vu/src/main/kotlin/jces1209/vu/page/boards/view/cloud/CloudScrumBoardPage.kt

```diff
@@ -0,0 +1,29 @@
+package jces1209.vu.page.boards.view.cloud
+
+import jces1209.vu.page.boards.sprint.SprintPage
+import jces1209.vu.page.boards.view.ScrumBoardPage
+import jces1209.vu.page.boards.view.SprintBoardComponent
+import org.openqa.selenium.WebDriver
+import java.net.URI
+
+class CloudScrumBoardPage(
+    driver: WebDriver, uri: URI
+) : CloudClassicBoardPage(driver, uri), ScrumBoardPage {
+
+    override fun goToBacklog(): SprintBoardComponent {
+        driver.navigate().to(
+            URI(
+                uri.scheme,
+                uri.authority,
+                uri.path,
+                uri.query + "&view=planning&issueLimit=100",
```

Comment:
Interesting. we reconstruct a direct link to backlog. I guess some users do have it bookmarked or navigate via browser history/autocompletion. I saw people usually go to their board and click the Backlog button. But that's a detail we can revisit in the future, not a big deal.